scripts/bert/finetune_classifier.py

```diff
@@ -34,68 +34,99 @@
 # under the License.
 # pylint:disable=redefined-outer-name,logging-format-interpolation
 
+import os
+import time
 import argparse
 import random
 import logging
+import warnings
 import numpy as np
 import mxnet as mx
 from mxnet import gluon
+import gluonnlp as nlp
 from gluonnlp.model import bert_12_768_12
 from bert import BERTClassifier
 from tokenizer import FullTokenizer
 from dataset import MRPCDataset, ClassificationTransform
 
-np.random.seed(0)
-random.seed(0)
-mx.random.seed(2)
-logging.getLogger().setLevel(logging.DEBUG)
-
 parser = argparse.ArgumentParser(description='BERT sentence pair classification example.'
                                              'We fine-tune the BERT model on MRPC')
-parser.add_argument('--epochs', type=int, default=3, help='number of epochs')
+parser.add_argument('--epochs', type=int, default=3, help='number of epochs, default is 3')
 parser.add_argument('--batch_size', type=int, default=32,
-                    help='Batch size. Number of examples per gpu in a minibatch')
-parser.add_argument('--test_batch_size', type=int, default=8, help='Test batch size')
-parser.add_argument('--optimizer', type=str, default='adam', help='optimization algorithm')
-parser.add_argument('--lr', type=float, default=5e-5, help='Initial learning rate')
+                    help='Batch size. Number of examples per gpu in a minibatch, default is 32')
+parser.add_argument('--dev_batch_size', type=int, default=8,
+                    help='Batch size for dev set, default is 8')
+parser.add_argument('--optimizer', type=str, default='adam',
+                    help='Optimization algorithm, default is adam')
+parser.add_argument('--lr', type=float, default=5e-5,
+                    help='Initial learning rate, default is 5e-5')
 parser.add_argument('--warmup_ratio', type=float, default=0.1,
-                    help='ratio of warmup steps used in NOAM\'s stepsize schedule')
-parser.add_argument('--log_interval', type=int, default=10, help='report interval')
-parser.add_argument('--max_len', type=int, default=128, help='Maximum length of the sentence pairs')
+                    help='ratio of warmup steps used in NOAM\'s stepsize schedule, default is 0.1')
+parser.add_argument('--log_interval', type=int, default=10, help='report interval, default is 10')
+parser.add_argument('--max_len', type=int, default=128,
+                    help='Maximum length of the sentence pairs, default is 128')
+parser.add_argument('--seed', type=int, default=2, help='Random seed, default is 2')
+parser.add_argument('--accumulate', type=int, default=None, help='The number of batches for '
```

Comment:
consider adding a message for the actual effective batch size based on this option, when it's set.

Reply:
Added